ui/apps/platform/src/Components/CheckboxSelect.tsx

```diff
@@ -42,9 +45,14 @@ function CheckboxSelect({
             onClick={onToggleClick}
             isExpanded={isOpen}
             isDisabled={isDisabled}
+            icon={toggleIcon}
         >
             {toggleLabel}
-            {selection && selection.length > 0 && <Badge isRead>{selection.length}</Badge>}
+            {selection && selection.length > 0 && (
+                <Badge isRead className="pf-v5-u-ml-sm">
```

Comment:
An observation, not a change request: an alternative to utility class is `Flex` layout:

```tsx
<Flex spaceItems={{ default: 'spaceItemsSm' }}>
    {toggleLabel}
    {selection && selection.length > 0 && (…)}
</Flex>
```

No badge, no space, no worries.

Reply:
@pedrottimark I'm a little confused about not having a badge. In other areas (e.g. Network Graph), we use badges to denote there are multiple selections for a checkbox dropdown. Isn't that something we want to preserve?

Reply:
Sorry to be cryptic. I meant if length is zero, conditional rendering omits the badge, and `spaceItems` does not apply, because only one flex item.